src/library-authoring/edit-library/LibraryEditPage.jsx

```diff
@@ -43,6 +45,7 @@ class LibraryEditPage extends React.Component {
         type: null,
         allow_public_learning: false,
         allow_public_read: false,
+        license: null,
```

Comment:
Noting a minor inconsistency: this is set to `''` in api.js.

Reply:
This is intentional since '' is a valid value for this field (it maps to All Rights Reserved), and if it's present and we render the dropdown component, that value will be cached, which will prevent the dropdown from updating properly when we set the library's values later.

Having it be null means I can check if the value is 'ready' before rendering the field, so I know I'm not letting the cache kick in before we have a valid value.